packs/iconics/samo-level-3.json

```diff
@@ -3663,24 +3663,7 @@
                         "key": "FlatModifier",
                         "selector": "skill-check",
                         "type": "proficiency",
-                        "value": {
-                            "brackets": [
-                                {
-                                    "end": 4,
-                                    "value": "@actor.level -2"
-                                },
-                                {
-                                    "end": 6,
-                                    "start": 5,
-                                    "value": "@actor.level -1"
-                                },
-                                {
-                                    "start": 7,
-                                    "value": "@actor.level"
-                                }
-                            ],
-                            "field": "actor|level"
-                        }
+                        "value": "match(when(lte(@actor.level, 4), \"@actor.level -2\"), when(btwn(@actor.level, 5, 6), \"@actor.level -1\"), when(gte(@actor.level, 7), \"@actor.level\"))"
```

Comment:
I would imagine that `@actor.level -2` without quotes or escaped quotes is the more correct one? Does math support a double resolve?